packages/react-components/src/components/chart/useChartsLegend.tsx

```diff
@@ -0,0 +1,72 @@
+import { InternalGraphicComponentGroupOption } from './types';
+import React, { useEffect, useState } from 'react';
+import { TableProps } from '@cloudscape-design/components/table/interfaces';
+import { GraphicComponentTextOption } from 'echarts/types/src/component/graphic/GraphicModel';
+import { SeriesOption } from 'echarts';
+import { LineSeriesOption } from 'echarts/types/src/chart/line/LineSeries';
+
+const useChartsLegend = ({
+  graphic,
+  series,
+}: {
+  graphic: InternalGraphicComponentGroupOption[];
+  series: SeriesOption[];
+}) => {
+  const legendColumnDefinition = {
+    id: 'Legends',
+    header: <div className='base-chart-legend-col-header'>Legends</div>,
```

Comment:
can we rename this column `properties` since its representing an asset prop in a chart

Reply:
I am not entirely sure what you are referring here.. is it he classname ?